docs/4.x/heatmap-session-recording/faq.md

```diff
@@ -244,3 +244,29 @@ console of your browser.
 ## How do I solve the issue of a partially rendered page with pictures or content missing?
 
 The screenshot for a Heatmap is taken when the page is officially loaded according to the browser. In some cases, the browser may consider the page fully loaded and rendered before all components are actually loaded.  Learn how to enable heatmaps once the page is fully loaded [in this FAQ](https://matomo.org/faq/heatmap-session-recording/faq_25413/).
+
+
+## How do I disable loading of CSS from database for Heatmap and Session Recording?
+
+Since the release of version **4.3.0** of Heatmap and Session Recording plugin, Matomo now tries to save the CSS internally in database by default to ensure the recording works even after the CSS link expires. However you can disable this feature by setting below setting in your config/config.ini.php.
+
+```php
+[HeatmapSessionRecording]
+load_css_from_db = 0
+```
+
+**Note**: This setting is enabled by default.
+
+## How do I fix CORS issue for Matomo to save CSS internally in database.
```

Comment:
```suggestion
## How do I fix CORS issue for Matomo to save CSS internally in database?
```

Also I really don't understand what this is referring to. What is the "CORS issue for Matomo"?

Reply:
so 

> If you got some time left, maybe check out https://languagetool.org/. It is an amazing FOSS spell/grammar/style-checker and it really helps to find those tiny typos one does all the time and that are easy to overlook. I also wrote a command-line tool that supports checking markdown files directly: https://pylanguagetool.lw1.at/

Thanks @Findus23

Reply:
Okay, So we are now attempting to save the CSS internally in database with release of new version of the plugin. However there can be a case where the CSS cannot be stored.
For example: We attempt to fetch the CSS content if `document.styleSheets[{index}].cssRules` fails, we fetch the content using a XHR request, but this request can be blocked due to CORS policy